src/split-panel/preferences-modal.tsx

```diff
@@ -16,13 +16,13 @@ import bottomPositionIconRefresh from './icons/bottom-icon-refresh';
 import { InternalBaseComponentProps } from '../internal/hooks/use-base-component';
 
 interface PreferencesModali18nStrings {
-  header: string;
-  cancel: string;
-  confirm: string;
-  positionLabel: string;
-  positionDescription: string;
-  positionBottom: string;
-  positionSide: string;
+  header?: string;
+  cancel?: string;
+  confirm?: string;
+  positionLabel?: string;
+  positionDescription?: string;
+  positionBottom?: string;
+  positionSide?: string;
 }
```

Comment:
I had two choices here

1. Add non-null assertion in the `split-panel/index.tsx` file
2. Mark them optional here

I picked the 2nd option because it is closer to runtime behavior (these properties may be missing)

Reply:
I'm happy with the 2nd option.